cmd/install/install.go

```diff
@@ -475,6 +477,25 @@ func hyperShiftOperatorManifests(opts Options) ([]crclient.Object, error) {
 		objects = append(objects, userCABundleCM)
 	}
 
+	hasCAInjectionCapability, err := mgmtClusterHasCAInjectionCapability()
```

Comment:
Actually, I think it won't hurt to always have the configmap and to mount it. So you can skip the detection part. That will result in a more consistent experience when rendering the install manifests. If the mgmt cluster doesn't have a CNO, then we just get an empty dir mounted.

Reply:
Ok, I removed this code out of their and its related code.